web-crawler/src/main/java/com/google/sps/webcrawler/WebCrawler.java

```diff
@@ -232,27 +232,20 @@ public void storeInDatabase(String candidateId, NewsArticle newsArticle) {
         .set("candidateId", datastore.newKeyFactory().setKind("Candidate").newKey(candidateId))
         .set("title", newsArticle.getTitle())
         .set("url", newsArticle.getUrl())
-        .set("content", excludeStringFromIndexes(combineContentInHtml(newsArticle.getContent())))
+        .set("content", excludeStringFromIndexes(newsArticle.getContent()))
         .set("abbreviatedContent", excludeStringFromIndexes(newsArticle.getAbbreviatedContent()))
         .build();
     datastore.put(newsArticleEntity);
   }
 
   /** 
    * Converts {@code String} to {@code StringValue} and excludes the data from indexes, to avoid
```

Comment:
I would elaborate on how this is specific to DataStore functionality (i.e. what does "indexes" mean in this context?)

Reply:
Will do!